share/spack/gitlab/cloud_pipelines/stacks/windows-vis/spack.yaml

```diff
@@ -0,0 +1,18 @@
+# Windows Visualization Stack
+# maintainers:
+# - John Parent (@johnwparent)
+# - Ryan Krattiger (@kwryankrattiger)
+
+spack:
+  view: false
+
+  include:
+  - $spack/share/spack/gitlab/cloud_pipelines/configs/win64/x86_64
```

Comment:
I think getting `--config-scope` working is better for now.

Reply:
Done, removed these files and restored `--config-scope`